bin/mv.js

```diff
@@ -53,83 +53,80 @@ function handleWebtaskMove(args) {
 }
 
 function moveWebtask(profile, name, target) {
-    var source = {
-        name: name,
-        container: profile.container,
-        profile: profile.name
-    };
+    var sourceWebtask;
 
-    if (equal(source, target)) {
+    if (equal({
+            name: name,
+            container: profile.container,
+            profile: profile.name
+        }, target)) {
         throw Cli.error.invalid('Webtasks are identical. Use a different target name, container or profile.');
     }
 
-    return read(profile, name)
-        .then(function(webtask) {
-            return copy(profile, webtask, target);
-        })
-        .then(function() {
-            return profile.removeWebtask({
-                name: name
-            });
-        });
-}
-
-function equal(source, target) {
-    return _.isEqual(source, {
-        name: target.name,
-        container: target.container || source.container,
-        profile: target.profile || source.profile
-    });
-}
-
-function read(profile, name) {
-    return profile.inspectWebtask({
-            name: name,
-            decrypt: true,
-            fetch_code: true
+    return profile.getWebtask({
+            name: name
         })
         .catch(function(err) {
             if (err.statusCode === 404) {
                 throw Cli.error.notFound('No such webtask: ' + Chalk.bold(name));
             }
             throw err;
+        })
+        .then(function(webtask) {
+            sourceWebtask = webtask;
+            return webtask.inspect({
+                decrypt: true,
+                fetch_code: true
+            });
+        })
+        .then(function(data) {
+            return copy(sourceWebtask, data, target);
+        })
+        .then(function() {
+            return sourceWebtask.remove();
```

Comment:
For now, let's leave this as-is, but down the line I think we could consider a `copy` variant triggered by command-line argument. Perhaps the default command would be `copy` and we would add a `--move` or `--remove` flag to signal move semantics.